src/pages/PublicProfile.vue

```diff
@@ -37,6 +37,7 @@ export default {
       isPlaceSearchEnabled,
       userRatingsByTransaction: [],
       userRatingsLoaded: false,
+      social: { instagram: 'Instagram Link', facebook: 'Facebook Link', twitter: 'Tiktok Link', website: 'Clubhouse Link' },
```

Comment:
@Digital4design Please put these in the language definition file: https://github.com/justinschuldt/syr-marketplace/blob/dev/src/i18n/source/user.yaml

Also, why is it `twitter: 'Tiktok Link', website: 'Clubhouse Link'`? Can you please update these to be consistent, if we telling the user to input tiktok and clubhouse links, we should name the internal properties to accurately reflect the data that is being captured. Please update the metadata property names and the css classes to match the labels.

Reply:
removed form file and add these social labels in teh user.yaml and used it to show labels